test/types/pino.test-d.ts

```diff
@@ -90,6 +91,7 @@ pino({
 });
 
 pino({ base: null });
+// @ts-expect-error
 if ("pino" in log) console.log(`pino version: ${log.pino}`);
```

Comment:
Not sure why it is inside the type check.
I cannot find the relevant code in js.